### PR TITLE
Variations - Remove separator between buttons and empty state screen

### LINKED ISSUE #38122: [Enhancement]: Variations - Remove separator between buttons and empty state screen
The line separating the buttons from the rest of the empty state is removed.
screenshot-one local-2023 05 04-14_04_47

### PR DESCRIPTION
**Pull request by octaedro**
### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

This PR removes the separator between buttons and the empty state screen.

screenshot-one local-2023 05 04-14_06_32

Closes #38122.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to Products > Add New > Product data. 
2. Select `Variable product`.
3. Create an attribute (or add a global attribute if you already have one). 
4. Press `Save attributes`.
5. Press `Variations` and verify that the separator between the buttons and the empty state screen is no longer visible.

### CODE REVIEW

File: plugins/woocommerce/client/legacy/css/admin.scss [hunk 1]
```diff
@@ -5505,13 +5505,15 @@ img.help_tip {
   * WooCommerce meta boxes
   */
 .wc-metaboxes-wrapper {
+	&:not(#variable_product_options_inner) {
```
Comment (mattsherman) [inline plugins/woocommerce/client/legacy/css/admin.scss]: Haven't spent much time looking into what the selector should be, but this isn't right. 

`#variable_product_options_inner` is a child of `.wc-metaboxes-wrapper`

**Review by mattsherman — CHANGES_REQUESTED** [review]
The separator is not shown when the variations list is shown.

Screenshot 2023-05-05 at 15 48 25

**Comment by octaedro** [comment]
Thank you @mattsherman for your comments.
I just fixed the bug you mentioned, could you take another look at this PR?

**Comment by octaedro** [comment]
Thank you @mattsherman for your comments.
I just fixed the bug you mentioned, could you take another look at this PR?

**Review by mattsherman — APPROVED** [review]
Works as expected. Nice job. :shipit:

**Pull request merged**